source/include/shadow.h

```diff
@@ -470,10 +548,14 @@ typedef enum ShadowStatus
  * uint16_t outLength = 0;
  * const char * pThingName = "TestThingName";
  * uint16_t thingNameLength  = ( sizeof( thingName ) - 1U );
```

Comment:
```suggestion
 * uint16_t thingNameLength  = ( sizeof( pThingName ) - 1U );
```

Reply:
Looking at this now, I see the example in the comments before was incorrect. sizeof() here will return the size of the pointer, when it's meant to be the size of the string/array. So I've made a different change.